bin/node/src/chain_spec.rs

```diff
@@ -226,9 +236,28 @@ fn system_properties(token_symbol: String) -> serde_json::map::Map<String, Value
     .collect()
 }
 
-pub fn config(
+pub fn devnet_config(chain_params: ChainParams, authorities: Vec<AuthorityKeys>)
+                     -> Result<ChainSpec, String> {
+    generate_chain_spec_config(chain_params,
+                               authorities.clone(),
+                               (0..authorities.len())
```

Comment:
Initialize this separately as `let stakers = ...` and pass to `generate_chain_spec_config`. I'll be more readable.

Reply:
WIll do